### PR TITLE
ZENKO-3365: update zenko-ui + add shell-ui

### PR DESCRIPTION
**Pull request by alexanderchan-scality**
**What does this PR do, and why do we need it?**
* update zenko-ui image
* deploy shell-ui dependency in tests
* update zenko deploy to read from deps.yaml

**Which issue does this PR fix?**

fixes ZENKO-3365

**Special notes for your reviewers**:

### CODE REVIEW

**Review by anurag4DSB — APPROVED** [review]

**Review by nicolas2bert — APPROVED** [review]

File: eve/main.yml [hunk 1]
```diff
@@ -20,6 +20,8 @@ models:
       OIDC_PASSWORD: 'end2end_password'
       OIDC_FIRST_NAME: 'hello'
       OIDC_LAST_NAME: 'world'
+      SHELL_UI_IMAGE: "registry.scality.com/playground/jbwatenberg/shell-ui:2.9-dev"
```
Comment (tcarmet) [inline eve/main.yml:23]: Please find a proper place for this image. `playground` is a place for us to play with images, not for images to be actually commited in our code.

I can help in setting up the proper place if needed, feel free to ping me for that.

Comment (tcarmet) [reply eve/main.yml:23]: cc: @JBWatenbergScality

Comment (terrydervaux) [reply eve/main.yml:23]: ticket created to take care of this issue -> [ZENKO-3405](https://scality.atlassian.net/browse/ZENKO-3405)

Comment (JBWatenbergScality) [reply eve/main.yml:23]: We also have one in metalk8s : https://github.com/scality/metalk8s/issues/3269

**Review by anurag4DSB — APPROVED** [review]

File: eve/workers/end2end/scripts/configs/zenkoversion.yaml [hunk 2]
```diff
@@ -1,74 +1,73 @@
...
     management:
       ui:
-        image: registry.scality.com/zenko-ui/zenko-ui
-        tag: '0.1.0'
+        image: '${ZENKO_UI_IMAGE}'
+        tag: '${ZENKO_UI_TAG}'
       api:
-        image: registry.scality.com/sf-eng/pensieve-api
-        tag:  '46e279b'
+        image: '${PENSIEVE_API_IMAGE}'
```
Comment (nicolas2bert) [inline eve/workers/end2end/scripts/configs/zenkoversion.yaml:14]: How these variables get populated? (I might be missing something obvious.)

Comment (alexanderchan-scality) [reply eve/workers/end2end/scripts/configs/zenkoversion.yaml:14]: the values are retrieved from `solution/deps.yaml` https://github.com/scality/Zenko/pull/1195/files#diff-a104984140e55eb80c2eea14bb0b3a260ed9ce9fcdc3b3c4335f0319265d5d89R14

**Comment by anurag4DSB** [comment]
There are 2 logical changes in this PR.
Is there a ticket to follow up on the changes we made on zenkoversions.yml. I see it's an auto-generated file: https://github.com/scality/Zenko/pull/1195/files#diff-9f6fa10f01612d716bcbf338754ee53446b2811984c908f9290b4f380eb452c2R2

Or can we change the description of the current ticket by adding the summary of the changes made to zenkoversions.yml which is not an autogenerated file

**Review by nicolas2bert — APPROVED** [review]

**Pull request merged**